Packages/Tracking/OpenXR/Runtime/Scripts/OpenXRLeapProvider.cs

```diff
@@ -91,7 +91,7 @@ private TrackingSource CheckOpenXRAvailable()
             if (XRGeneralSettings.Instance != null &&
                 XRGeneralSettings.Instance.Manager != null &&
                 XRGeneralSettings.Instance.Manager.activeLoader != null &&
-                XRGeneralSettings.Instance.Manager.activeLoader.name == "Open XR Loader" &&
+                XRGeneralSettings.Instance.Manager.activeLoader.name.Replace(" ", "") == "OpenXRLoader" && // Some Unity versions have spaces and some don't so remove the spaces
```

Comment:
I think we should be using `XRGeneralSettings.Instance.Manager.ActiveLoaderAs<OpenXRLoaderBase>() != null` instead which actively tries to get it as an OpenXRLoader and will return null if it's not that type. We need to use `OpenXRLoaderBase` as there is `OpenXRLoader` and `OpenXRLoaderNoPreInit` which both are possible (not sure what uses the no pre-init version though.

Reply:
Oh yeah that would be much cleaner - i'll take a look at that now

Reply:
Oh yeah that would be much cleaner - i'll take a look at that now

Reply:
If the base doesn't work, we might need to test for both `OpenXRLoader` and `OpenXRLoaderNoPreInit` seperately.

Reply:
It works great, and falls back to LeapC if not using OpenXR as the XR Plugin